raiden/tests/unit/test_blockchain_events_handler.py

```diff
@@ -51,3 +57,33 @@ def test_create_update_transfer_state_change_unknown_channel(container, update_t
         chain_state=container.chain_state, event=update_transfer
     )
     assert state_change is None
+
+
+@pytest.fixture
+def channel_closed(container, event_data):
```

Comment:
I'd rename this: `channel_closed` -> `channel_closed_event`, same for all other event fixtures